rails/benchmarks/actioncable_redis.rb

```diff
@@ -0,0 +1,21 @@
+require 'bundler/setup'
+require 'rails'
+
+require_relative 'support/benchmark_rails.rb'
+require_relative 'support/actioncable_helpers.rb'
+
+REDIS_FAYE = Faye::WebSocket::Client.new("ws://127.0.0.1:4002/")
+
+REDIS_DB_URL = ENV["REDIS_PORT_6379_TCP_ADDR"]
+
+Benchmark.rails("actioncable_redis", time: 10) do
+  redis_config =  { adapter: 'redis', url: REDIS_DB_URL }.with_indifferent_access
+  with_puma_server(ActionCable.server, 4002, redis_config) do |port|
+    sleep
```

Comment:
I wrote a dummy server https://github.com/bf4/active_model_serializers/blob/4c79a5e24d667ea1974d74119d758658369ead16/bin/serve_dummy for https://github.com/rails-api/active_model_serializers/pull/1412 which I"m working on for benchmarking AMS https://github.com/rails-api/active_model_serializers/pull/1393  (looking around here for ideas ;)